.gitignore

```diff
@@ -297,3 +297,6 @@ dists/emscripten/emsdk-*
 #Ignore Atari/FreeMiNT files
 scummvm.gtp
 scummvm.ttp
+
+#Ignore Dear imgui file
+imgui.ini
```

Comment:
I really wonder how this file is created. In a sense, the current directory could potentially be non-writeable.

Is it possible to make imgui store its configuration in our ConfMan?

Reply:
Yes, you're right, I should do that.
You can configure imgui to specify where to write the imgui.ini.
I add it to my Todo list.